tests/app/authentication/test_database_session_manager.py

```diff
@@ -16,10 +16,9 @@ class TestSessionManager(AppContextTestCase):
 
     def setUp(self):
         super().setUp()
-        self.database = Mock(Database(driver='sqlite',
-                                      database_name='',
-                                      setup_attempts=1,
-                                      setup_retry_delay=0))
+        self.database = Mock(Database(application=self._app,
+                                      driver='sqlite',
+                                      database_name=''))
```

Comment:
Should this not have some name something like test_db?

Reply:
Its mocked so it makes no difference